docs/install_linux.md

```diff
@@ -256,20 +291,31 @@ Upon first run, the bot will automatically create the database tables needed to
 If the bot runs without any errors, you have had success so far!
 
 You can test simple functionality by running the `ping` command (With your desired prefix) in a text channel that the bot can see. If it answers "Pong!", then congratulations, *you have successfully set up WildBeast!*
+
+##RethinkDB Dashboard
+**The following information is for those who need to access the RethinkDB web interface to create, delete or edit the database contents.**
+
+Should you need to access the RethinkDB dashboard remotely to create, delete or edit the information stored within, You may create a tunnel via SSH to forward traffic from your browser to the server or run RethinkDB using the `--bind all` option with firewall rules to only allow your IP address to connect, this guide will not provide the required information to do this.
```

Comment:
A few readability improvement suggestions here.

- I'd say put a dot after `information stored within`, or we have a one-sentence paragraph that is clunky as heck to read.
- Also put a dot after `address to connect` to improve readability. Change `this` to capital form accordingly.